lib/ruby_ui/masked_input/masked_input.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module RubyUI
+  class MaskedInput < Base
+    def view_template
+      Input(type: "text", **attrs)
```

Comment:
@stephannv , I like the way you did it here. It's using the Input component that we already have, which makes MaskedInput compatible with our forms.

We will have to find a way to mark the dependencies so the component generator could be smart enough to eject all dependencies correctly. But we can do that later